source/documentation/io/rdf-input.md

```diff
@@ -137,23 +134,35 @@ The parsers log to a logger called `org.apache.jena.riot`.  To avoid `WARN`
 messages, set this to `ERROR` in the logging system of the application.
 
 ## StreamManager and LocationMapper
-    
+
+Operations to read RDF data can be redirected to local copies and to other URLs.
+This is useful to provide local copies of remote resources.
+
 By default, the `RDFDataMgr` uses the global `StreamManager` to open typed
-InputStreams.  This is available to applications via `RDFDataMgr.open` as well as directly
-using a `StreamManager`.
+InputStreams.  The `StreamManager` can be set using the `RDFParser` builder:
+
+        // Create a copy of the global default StreamManager.
+        StreamManager sm = StreamManager.get().clone();
+        // Add directory "/tmp" as a place to look for files
+        sm.addLocator(new LocatorFile("/tmp"));
+
+        RDFParser.create()
+            .streamManager(sm)
+            .source("data.ttl")
+            .parse(...);
 
-The `StreamManager` is chosen based on the `Context` object for the
+It can also be set in the chosen based on the `Context` object for the
```

Comment:
I think it's a bit confusing. Should we define what's the chosen thing/subject is here?